Update README for 8.0 (#30126)
PrestaShop/PrestaShop: README.md

 
For more information on our branch system, read our guide on [installing PrestaShop for development][install-guide-dev].

> The first stable version of PrestaShop 8.0, was released on October 26th, 2022. Further updates have been released since then. Learn more about it on [the Build devblog](https://build.prestashop.com/tag/8.0/).

Reviewer: ```suggestion
The first stable version of PrestaShop 8.0, was released on October 26th, 2022. Further updates have been released since then. Learn more about it on [the Build devblog](https://build.prestashop-project.org/tag/8.0/).
```

[contributing-md]: CONTRIBUTING.md

> [contributing-tutorial]: https://devdocs.prestashop.com/8/contribute/

Reviewer: ```suggestion
[contributing-tutorial]: https://devdocs.prestashop-project.org/8/contribute/
```

[crowdin]: https://crowdin.net/project/prestashop-official

> [getting-started]: https://docs.prestashop-project.org/-documentation/getting-started

Reviewer: ```suggestion
[getting-started]: https://docs.prestashop-project.org/1.7-documentation/getting-started
```

this link doesn't work
